generators/firechip/src/main/scala/TargetConfigs.scala

```diff
@@ -47,13 +47,17 @@ class WithUARTKey extends Config((site, here, up) => {
      nRxEntries = 256))
 })
 
+class WithSerial extends Config((site, here, up) => {
```

Comment:
Shouldn't there be other parameters for TSI (at least bitwdith?) rolled into this key?

Reply:
Can we leave that to future work? It doesn't look like testchipip exports a clean way to configure that yet.

Reply:
Sure. We should file an issue on testchipip so we don't lose track of this feature request.

Reply:
https://github.com/ucb-bar/testchipip/issues/63